eras/mary/impl/src/Cardano/Ledger/Mary/Value.hs

```diff
@@ -177,13 +176,10 @@ instance Group (MultiAsset c) where
     MultiAsset (canonicalMap (canonicalMap ((-1 :: Integer) *)) m)
 
 instance Crypto c => DecCBOR (MultiAsset c) where
-  decCBOR = decodeMultiAssetMint
-
-instance Crypto c => EncCBOR (MultiAsset c) where
-  encCBOR = encodeMultiAssetMaps
+  decCBOR = decodeMultiAsset decodeIntegerBounded64
```

Comment:
I think it has moved in the newtype declaration above  
```
newtype MultiAsset c = MultiAsset (Map (PolicyID c) (Map AssetName Integer))
  deriving (Show, Generic, ToJSON, EncCBOR)
```

since `encodeMultiAssetMaps` was only doing `encCBOR m`

Reply:
Yes. It can be derived with GeneralizedNewtypeDeriving, no need to specify it manually